File: src/main/java/com/codingbat/warmup1/MonkeyTrouble.java
package com.codingbat.warmup1;

public class MonkeyTrouble {

  public static boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
    // if both monkeys are smiling, we are in trouble
    // if neither monkey is smiling, we are in trouble
    return (aSmile && bSmile) || !(aSmile || bSmile);
  }
}

File: src/main/kotlin/com/codingbat/warmup1/MonkeyTrouble.kt
package com.codingbat.warmup1

fun monkeyTrouble(aSmile: Boolean, bSmile: Boolean): Boolean {
  // if both monkeys are smiling, we are in trouble
  // if neither monkey is smiling, we are in trouble
  return (aSmile && bSmile) || !(aSmile || bSmile)
}
